### DISCUSSION TITLE
Cannot Connect to Google Sheets

### DISCUSSION THREAD

**Discussion by agschech** [Budibase/budibase#11231] [Help] [answered]
Hi all,
When I connect to Google sheets and then authenticate with my google account, approve permissions etc. I am returned to the Budibase data tab as if nothing happened.  I am not prompted with the URL or anything like I see in the how-to.

I am on the free version of Budibase and running Edge as my browser.

Can anyone suggest how to possibly address this?

Edit: I have followed all the instructions here with no luck yet: https://docs.budibase.com/docs/sso-with-google

When I created the credentials, I received a dialog mentioning it could take 5 minutes to a few hours for all changes to go into effect. Perhaps I just need to check back later?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @agschech,

Have you checked again recently, is this still not working for you?

**Reply by agschech** [reply]
It is now working, thank you for checking in!

**Reply by ConorWebb96** [reply]
Happy to hear it!
